scripts/ci/initialize_mysql.sh

```diff
@@ -30,11 +30,13 @@ function bootDB {
   fi
 
   if [ "$db" = "postgres" ]; then
-    launchDB="(docker-entrypoint.sh -c max_connections=300 &> /var/log/postgres-boot.log) &"
-    testConnection="psql -h localhost -U $POSTGRES_USER -c '\conninfo' &>/dev/null"
+    #launchDB="(/postgres-entrypoint.sh -c max_connections=300 &> /var/log/postgres-boot.log) &"
+    launchDB="(POSTGRES_PASSWORD=password POSTGRES_USER=root /postgres-entrypoint.sh postgres -c max_connections=300 &> /var/log/postgres-boot.log) &"
+    #testConnection="psql -h localhost -U $POSTGRES_USER -c '\conninfo' &>/dev/null"
+    testConnection="PGPASSWORD=password psql -h localhost -U root -c '\conninfo'"
   elif [[ "$db" == "mysql"* ]]; then
     chown -R mysql:mysql /var/run/mysqld
-    launchDB="(MYSQL_USER='' MYSQL_ROOT_PASSWORD=$MYSQL_PASSWORD /entrypoint.sh mysqld &> /var/log/mysql-boot.log) &"
+    launchDB="(MYSQL_USER='' MYSQL_ROOT_PASSWORD=$MYSQL_PASSWORD /mysql-entrypoint.sh mysqld --max_allowed_packet=256M  &> /var/log/mysql-boot.log) &"
```

Comment:
Found it odd that the max_allowed_packet now needs set. I wonder how this was configured before- apparently this has been required to run migration tests due to a pxc(?) bump prior to this work.

Reply:
It looks like this is how it was done before? https://github.com/pivotal/tas-runtime/commit/e0c34acdcc62b998a385a336e900a4567462e5d1

Reply:
So I'm rescinding my comment :)